applications/luci-app-package-manager/htdocs/luci-static/resources/view/package-manager.js

```diff
@@ -661,6 +664,7 @@ function handleReset(ev)
 function handleInstall(ev)
 {
 	var name = ev.target.getAttribute('data-package'),
+	    installstyle = ev.target.getAttribute('data-action'),
```

Comment:
would call the variable cmd.

Reply:
I wanted to name it differently than 'cmd' for clarity, as handlePkg function already has 'cmd'

Reply:
I will change it to installcmd